src/library_memfs.js

```diff
@@ -274,6 +274,9 @@ mergeInto(LibraryManager.library, {
       },
 
       // Writes the byte range (buffer[offset], buffer[offset+length]) to offset 'position' into the file pointed by 'stream'
+      // canOwn: A boolean that tells if this function can take ownership of the passed in buffer from the subbuffer portion
+      //         that the typed array view 'buffer' points to. The underlying ArrayBuffer can be larger than that, but
+      //         canOwn=true will not take ownership of the portion outside the bytes addressed by the view.
```

Comment:
Please clarify this a little more, as taking ownership of a range is not immediately obvious. Maybe something like "what actually happens is that `canOwn` means we can avoid a copy - we just use the view given us into the existing buffer. Since we just use that view, we don't notice anything outside that in the rest of the buffer."